src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -17,6 +17,10 @@ describe('<CourseRunEnrollment />', () => {
     sentry_dsn: 'the sentry dsn',
   };
 
+  const staticProps = {
+    signupUrl: 'https://demo.richie.education/singup',
```

Comment:
```suggestion
    signupUrl: 'https://demo.richie.education/signup',
```

Reply:
Also, I could not find where this prop is used.

Reply:
Yes my bad I forget to remove it.